backend/docs/source/visualizations.md

```diff
@@ -0,0 +1,29 @@
+# Visualizations in Clowder
+
+When browsing Clowder, there are two primary ways users can see aspects of a file or dataset without downloading it:
+
+- Thumbnails, which are lightweight static images that appear in grid views such as search results; and
+- Visualizations, which appear on individual resource pages and trigger JavaScript modules to create more specific
+  preview components such as map views or interactive widgets.
+
+Both of these can render actual bytes stored in the system, but Visualizations support more complex behavior. Typically,
+both Thumbnails and Visualizations are generated by extractors listening for new file uploads.
+
+## Thumbnails
+
+To programatically add a thumbnail to a file or dataset:
+
+1. POST an image to the `api/v2/thumbnails` endpoint to upload it to the system. The response will include
+   a `thumbnail_id`.
+2. Associate the thumbnail with one or more resources by issuing a PATCH request
+   to `api/v2/files/:file_id/thumbnails/:thumbnail_id` or `api/v2/datasets/:dataset_id/thumbnails/:thumbnail_id`.
```

Comment:
Typo. It will be

api/v2/files/:file_id/thumbnail/:thumbnail_id` or `api/v2/datasets/:dataset_id/thumbnail/:thumbnail_id`